definitions/apm-application/definition.yml

```diff
@@ -11,3 +11,22 @@ compositeMetrics:
 configuration:
   entityExpirationTime: QUARTERLY
   alertable: true
+
+relationships:
+  # Select entityGuid, peer.hostname from Span where peer.hostname is not null and category =
+  # 'datastore'
+  # ExtService in this case is a DB
+  - provider: newrelic
+    eventType: Span
```

Comment:
Would it make sense to make `eventType` an array?

Reply:
maybe? I'm not sure. The attributes would possibly/likely change as well so i'm unsure of how useful it would really be?

Reply:
no harm either right? Also better mimics what nrql can do

Reply:
done!